ingestion-beam/pom.xml

```diff
@@ -10,10 +10,15 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <beam.version>2.8.0</beam.version>
         <exec.mainClass>com.mozilla.telemetry.Sink</exec.mainClass>
         <!-- Workaround for https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=911925 -->
         <argLine>-Djdk.net.URLClassPath.disableClassPathURLCheck=true</argLine>
+
+        <!-- Keep these dependency versions in sync with those pulled in by beam;
+             check https://mvnrepository.com/artifact/org.apache.beam -->
+        <beam.version>2.8.0</beam.version>
+        <google-cloud.version>1.32.0</google-cloud.version>
+        <jackson.version>2.9.5</jackson.version>
```

Comment:
This should also prevent dependabot from trying to update these. I think it's not able to parse back through the logic of versions declared via property references.